Source/Fuse.Nodes/Node.uno

```diff
@@ -133,5 +133,28 @@ namespace Fuse
 			return this;
 		}
 
+		//We want `: Behavior` here. This is an attempt to workaround a Uno/DotNet issue on some platforms
```

Comment:
I didn't isolte a standalone test-case and thus didn't record an issue. Shall I remove this comment, or what shall I do instead?